runner/ansible/vars/gcp/10-default.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+expected_core:
+  "1.1.1": "20000"
+  "1.1.2": "" # unkown value, default may apply
+  "1.1.3": "20"
+  "1.1.4": "60"
+  "1.1.5": "10"
+  "1.1.6": "udpu"
+  "1.1.7": "2"
+  "1.1.8": "1"
+  "1.2.1": "true"
+  "1.2.2.fence_gcpstonith": "300" # new variable
```

Comment:
Not harmful, but this variable won't do anything by now. I would be for removing it, and put it once we have the test updated (or a new one added most probably), so the new PR will make more sense

Reply:
Done